WPMedia/fusion-news-theme-blocks
# [THEMES-1374] Implemented logical properties for search-results list and results-list

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

Update padding, margin, and border properties. The following conversions must be done:

*-top and *-bottom need to be changed to *-block

*-left and *-right need to be changed to *-inline

height and width will need to be set to block-size and inline-size respectively

## Jira Ticket

- [THEMES-1374](https://arcpublishing.atlassian.net/browse/THEMES-1374)

## Acceptance Criteria

1. The above logical properties are applied and tested on the list blocks.
2. Storybook is updated with RTL.

## Test Steps

1. Checkout this branch `git checkout THEMES-1374`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/search-results-list-block,@wpmedia/results-list-block`
4. Set "textDirection": "rtl" in the "siteProperties" object of the blocks.json
5. Check the styles for RTL matching to Figma designs

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1374]: https://arcpublishing.atlassian.net/browse/THEMES-1374?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
